dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/icon/IconControllerTest.java

```diff
@@ -65,6 +67,8 @@ void shouldCreateCustomIconWhenFileResourceExist() throws IOException {
                     + createFileResource()
                     + "', 'keywords':"
                     + keywords
+                    + ", 'custom' :"
+                    + custom
                     + "}")
             .content(HttpStatus.CREATED)
```

Comment:
You should be able to use a text block and create a more readable `key` with something like
```
    String key =
        """
{'key': '%s','description': '%s','fileResourceUid': '%s','keywords': %s}"""
            .formatted(iconKey, description, createFileResource(), keywords);
    JsonWebMessage message =
        POST("/icons/", key).content(HttpStatus.CREATED).as(JsonWebMessage.class);
```